README.md

```diff
@@ -1 +1,83 @@
-# to-do-list_with-React
\ No newline at end of file
+# Project Name
+
+Todo list 
+
+# Description the project.
+
+Todo list is apps that saves all the input task of your choice and you can edit or delete them  
+
+## Built With
+
+- html/css/js/React.
+- following linters standards.
+- following github flows standards.
+
+## Getting Started
+
+**Todo list app created with React**
+
+---
+
+To get a local copy up and running follow these simple example steps.
+follow>>>>>>>>>
+
+### Setup
+
+to use the project Leaderboard please follow :
+
+> Open GitHub and go to my GitHub repository name: To-do-list-with-React clone.
+
+> Click “Code” and copy the given URL.
+
+> click code and copy the URL.
+
+> click code and copy the URL.
+
+> Open “Git Bash” and change the current working directory to the location where you want the cloned directory.
+
+> Type git clone in the terminal, paste the URL you copied earlier, and press “enter” to create your local clone.
+
+> Syntax:
+
+$ git clone {repository URL}
+
+### Usage
+
+> open your directory Tv-show/
+
+\_use vs-code or any IDE to view line code .
+
+\_use you preferd browser to run the file .html to get project website.
+to run in terminal / git :
+
+
+$npm install
+
+---
+
+$ npm start
+
+---
+
+## Authors
+
+👤 **Said laasri**
+
+- GitHub: [@Said-laasri](https://github.com/Said-laasri)
+- Twitter: [@saidlaasr](https://twitter.com/saidlaasr)
+- LinkedIn: [SAID LAASRI](https://www.linkedin.com/in/said-laasri-8a4367172/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+thanks:
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+- Inspiration
+- etc
```

Comment:
It would be nice to mention the original author of the work,
you can mention his [Github](https://github.com/Ibaslogic)